examples/step-64/step-64.cu

```diff
@@ -298,16 +340,18 @@ namespace Step64
     AffineConstraints<double>                          constraints;
     std::unique_ptr<HelmholtzOperator<dim, fe_degree>> system_matrix_dev;
 
-    // Since all the operations in the solve functions are executed on the
-    // graphic card it is necessary for the vectors used to store their values
+    // Since all the operations in the `solve()` function are executed on the
+    // graphics card, it is necessary for the vectors used to store their values
     // on the GPU as well. LinearAlgebra::distributed::Vector can be told which
     // memory space to use. There is also LinearAlgebra::CUDAWrappers::Vector
     // that always uses GPU memory storage but doesn't work with MPI. It might
     // be worth noticing that the communication between different MPI processes
     // can be improved if the MPI implementation is CUDA-aware and the configure
-    // flag DEAL_II_MPI_WITH_CUDA_SUPPORT is enabled.
+    // flag `DEAL_II_MPI_WITH_CUDA_SUPPORT` is enabled. (The value of this
+    // flag is automatically determined at the time you call `cmake` when
+    // installing deal.II.)
```

Comment:
No, there is no autodetection. By default, it is always `OFF` and the user has to enable it manually.